workflow-bot-app/src/main/java/com/symphony/bdk/workflow/logs/LogsStreamingService.java

```diff
@@ -0,0 +1,58 @@
+package com.symphony.bdk.workflow.logs;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import org.springframework.web.servlet.mvc.method.annotation.SseEmitter;
+
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Service
+@Slf4j
+public class LogsStreamingService {
+  private final Map<String, SseEmitter> emitters = new HashMap<>();
+
+  public void subscribe(String clientId, SseEmitter emitter) {
+    log.debug("subscribe [{}] to sse.", clientId);
+    this.emitters.put(clientId, emitter);
+  }
+
+  public void broadcast(long timestamp, String level, String logger, String data) {
+    List<String> failedConnections = new ArrayList<>();
+    emitters.forEach((key, value) -> {
+      SseEmitter.SseEventBuilder event = SseEmitter.event()
+          .id(DateTimeFormatter.ISO_INSTANT.format(Instant.ofEpochMilli(timestamp)))
+          .data(String.format("[%s] %s - %s", level, logger, data))
+          .name("WDK SSE");
```

Comment:
name should just be `message` so that the standard `EventSource.onMessage()` listener can be used instead of having to construct a custom listener like so: `EventSource.addEventListener("WDK SSE", (e) => {})`

Reply:
sure, good to know